app/src/test/java/com/meewii/rateconverter/business/ExchangeRateRepositoryTest.kt

```diff
@@ -37,9 +33,27 @@ class ExchangeRateRepositoryTest {
   @Mock lateinit var userInputRepositoryMock: UserInputRepository
```

Comment:
I highly recommend to use Mockk instead of mockito :) With the latest developments and high adoption rate of coroutines, I truly believe that Mockk will be industry standard in the upcoming months

Reply:
Was hesitating to do that in the first place. It's done now. Thanks.